app/controllers/api/service_templates_controller.rb

```diff
@@ -50,6 +50,10 @@ def schedules_query_resource(object)
       object.miq_schedules
     end
 
+    def delete_resource_schedules(_parent, type, id, data)
```

Comment:
I think this method should go to app/controllers/api/subcollections/schedules.rb

Reply:
I couldn't include that mixin here since the relation works differently for ServiceTemplates than it previously did.